scripts/run_app.sh

```diff
@@ -50,6 +49,8 @@ if [ ! -s "static" ]; then
   exit 1
 fi
 
+$DIR/run_translate_schemas.sh
```

Comment:
Previous comment seems to have got lost; will this execute when deploying to EB via Jenkins?